CHANGELOG

```diff
@@ -105,6 +105,7 @@
 * Stripe: Add shipping address [jcreiff] #4539
 * PayuLatam: Add extra1, extra2, extra3 fields [jcreiff] #4550
 * Paysafe: Add fundingTransaction object [jcreiff] #4552
+* Shift4: Addition of required fields (recurring_pmt_num, recurring_pmt_count) to GSFs on MerchantE - Active Merchant [ali-hassan] #4553
```

Comment:
Hey @ali-hassan, can you move this to the bottom of this running list? Thanks.

Reply:
@naashton  done